test-utils/testlib/src/runtime_utils.rs

```diff
@@ -36,8 +36,8 @@ pub fn get_runtime_and_trie_from_genesis(
     let trie_update = TrieUpdate::new(trie.clone(), MerkleHash::default());
     let (store_update, genesis_root) = runtime.apply_genesis_state(
         trie_update,
-        &genesis_config.accounts,
-        &genesis_config.validators,
+        &genesis_config.accounts.iter().map(|account_info| (account_info.account_id.clone(), account_info.public_key.clone(), account_info.amount)).collect::<Vec<_>>(),
```

Comment:
This seems like a change unrelated to u128.

Reply:
This makes us output u128 in JSON in the format that JavaScript can understand (e.g. hex). This just spillover of re-org of GenesisConfig to allow it.